fastify.js

```diff
@@ -131,7 +129,6 @@ function fastify (options) {
   options.logger = logger
   options.genReqId = genReqId
   options.requestIdHeader = requestIdHeader
-  options.querystringParser = querystringParser
```

Comment:
It need to be deprecate in one release cycle instead of remove it directly.

Reply:
There is no change in functionality with this PR.

Reply:
I may read the code wrongly, it is still passing the same option.
But, it pass to another place.